shiptendance/views/show.ejs

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title></title>
+</head>
+<body>
+  <h1><%= pirate.name %></h1>
```

Comment:
Try to use a partial next time so you are not reusing the same code in each of your views.